### ISSUE TITLE
Why the color is different from before?

### ISSUE THREAD

**Issue by jerrydev** [jonschlinkert/sublime-monokai-extended#58]
Yesterday, when I'm coding, I found the color is changed. It's a unexpected change.

![before](https://cloud.githubusercontent.com/assets/5993771/11989032/0cfffecc-aa32-11e5-954c-5c7bfa0eab20.png)
![after](https://cloud.githubusercontent.com/assets/5993771/11989035/12b0be38-aa32-11e5-8887-35139ca8e1f5.png)

**Comment by SuperPaintman** [comment]
Use Monokai Origin

**Comment by jonschlinkert** [comment]
Sorry about that, I'll look into it and see what happened.

**Comment by karolyi** [comment]
+1 for python source highlighting (and probably more too):
original Monokai:
![image](https://cloud.githubusercontent.com/assets/987055/12010310/451f1bfa-aca1-11e5-8aa7-d6fe24e99716.png)

Monokai Extended:
![image](https://cloud.githubusercontent.com/assets/987055/12010312/6592be1e-aca1-11e5-8f81-6de03ffca00e.png)

`def` and `class` keywords are in different color.
more to come, probably ...

**Comment by ParadiseDS** [comment]
+1 for php highlighting
and the monokai bright cannot highlight under diff syntax

**Comment by jonschlinkert** [comment]
Yeah it shouldn't have changed. I'll revert whatever caused the issue.

**Comment by karolyi** [comment]
Package Control has upgraded Monokai Extended to 2015.12.29.22.12.53

the syntax is still not fixed

**Comment by OmeGak** [comment]
+1 for Python syntax highlight

**Comment by gerardroche** [comment]
PHP too.

At least one of the offending pr's is https://github.com/jonschlinkert/sublime-monokai-extended/pull/52

A diff between https://github.com/jonschlinkert/sublime-monokai-extended/commit/e99726ebe785dfb6352f175dd7decb363c3a05b0 and https://github.com/jonschlinkert/sublime-monokai-extended/commit/9a99355f6561c83a566b701ca30f41806c55b098

![diff](https://cloud.githubusercontent.com/assets/44148/12300181/475eb1ea-ba12-11e5-9bc1-9cbcc5746855.png)

Reverting causes some big conflicts.

**Comment by karolyi** [comment]
Package Control: Upgraded Monokai Extended to 2016.01.31.22.05.34

It seems the problem is solved. thx!

**Comment by OmeGak** [comment]
Keeps looking wrong in Python in 2016.01.31.22.05.34.

![screenshot from 2016-02-01 11 21 48](https://cloud.githubusercontent.com/assets/716307/12714720/a478d4b4-c8d6-11e5-8874-1d3cbcc5ac27.png)

**Comment by karolyi** [comment]
Interesting, mine is fixed:

![image](https://cloud.githubusercontent.com/assets/987055/12714882/6778e986-c8d7-11e5-8497-15b0c131ba3c.png)

**Comment by karolyi** [comment]
According to the latest merged pull request, the version installed must have fixed the issue:

https://github.com/jonschlinkert/sublime-monokai-extended/commit/6de4a667aa915cad33b8d1ec239138db42b2ca99

**Comment by OmeGak** [comment]
Thanks! I realized I was using the automatically converted Monokai Extended from Sublime Linter instead, which doesn't get updated automatically. I simply had to select a color scheme from Monokai Extended ones and Sublime Linter overwrote the old one.

**Comment by jonschlinkert** [comment]
I'm going to close since it sounds like this is fixed. However, we can reopen if others report that it's not. thanks for the feedback

**Comment by ice9js** [comment]
PHP syntax highlighting still looks like it's broken to me.

**Comment by jonschlinkert** [comment]
Still? Or as a result of https://github.com/jonschlinkert/sublime-monokai-extended/issues/62?

**Comment by gerardroche** [comment]
I just checked against build 3103 back to 3095 and it is still messed up.

It has been broken until back to commit https://github.com/jonschlinkert/sublime-monokai-extended/commit/e99726ebe785dfb6352f175dd7decb363c3a05b0 which is ok.

**Comment by jonschlinkert** [comment]
ok good to know, thanks for the detail

**Comment by gerardroche** [comment]
This is still an issue, at least for PHP. 

There are several issues. 

One is that, from what understand, the theme is generated from the `.JSON-tmTheme` files.  I didn't know this so when I contributed a [patch](https://github.com/jonschlinkert/sublime-monokai-extended/pull/42) I don't modify those files. I edited the `.tmTheme` file directly. 

In a later pull request my changes were overwritten, as I think many other changes, see https://github.com/jonschlinkert/sublime-monokai-extended/pull/52 

That pull request says "adds support for ~~ strike-through text" but look at the massive changes it made. It changed all kind of things. Any previous changes that didn't use the JSON file to generate the theme have been lost.

Look at the JSON files modifications in that pull request, they only add changes to the markup strikout, e.g.:

```
 +            "name": "Markup: strike", 
+            "scope": "markup.strike", 
+            "settings": {
+                "fontStyle": "", // Or add `strike`, but it doesn't support
+                "foreground": "#cc4273"
+            }
+        }, 
+        {
```

Then look at the changes to the `tmTheme` files, big sweeping changes.
